aea/cli/scaffold.py

```diff
@@ -155,3 +163,49 @@ def _scaffold_item(click_context, item_type, item_name):
         )
     except Exception as e:
         raise click.ClickException(str(e))
+
+
+def _scaffold_dm_handler(click_context):
+    """Add a scaffolded decision maker handler to the project and configuration."""
+
+    ctx = cast(Context, click_context.obj)
+    existing_dm_handler = getattr(ctx.agent_config, "decision_maker_handler")
+
+    # check if we already have a decision maker in the project
+    if existing_dm_handler != {}:
+        raise click.ClickException(
+            "A decision maker handler specification already exists. Aborting..."
+        )
+
+    try:
+        agent_name = ctx.agent_config.agent_name
+        click.echo(
+            "Adding decision maker scaffold to the agent '{}'...".format(agent_name)
+        )
+
+        # create the file name
+        dest = Path("decision_maker.py")
+        dotted_path = ".decision_maker"
+        class_name = "DecisonMakerHandler"
```

Comment:
loading not working like that...

Reply:
typo on `DecisonMakerHandler`, it should be "DecisionMakerHandler"